core/src/html/layout.rs

```diff
@@ -104,13 +104,8 @@ pub struct LayoutContext<'a, 'gc> {
     /// should be used.
     bounds: Option<BoxBounds<Twips>>,
 
-    /// The exterior bounds of all laid-out text, including left and right
-    /// margins.
-    ///
-    /// None indicates that no bounds have yet to be calculated. If the layout
```

Comment:
Is the part about `None` not correct anymore? (what's the new meaning of `None`?)

Reply:
Removed too much :D should be good now